python_modules/libraries/dagster-airlift/kitchen-sink/kitchen_sink/dagster_defs/retries_configured.py

```diff
@@ -0,0 +1,57 @@
+import requests
+from dagster import AssetExecutionContext, asset
+from dagster._core.storage.tags import (
+    AUTO_RETRY_RUN_ID_TAG,
+    MAX_RETRIES_TAG,
+    PARENT_RUN_ID_TAG,
+    RETRY_NUMBER_TAG,
+    WILL_RETRY_TAG,
+)
+from dagster_airlift.in_airflow.base_asset_operator import build_dagster_run_execution_params
+from dagster_airlift.in_airflow.gql_queries import TRIGGER_ASSETS_MUTATION
+
+
+# Asset that simulates having run retries activated (so that we don't have to stand up non-sqlite-storage)
```

Comment:
we actually just enabled run retries with sqlite storage if that simplifies anything here https://github.com/dagster-io/dagster/pull/26581

Reply:
cool - I'll see if I can simplify things here in a followup